.github/workflows/release-myget.yml

```diff
@@ -0,0 +1,31 @@
+name: Release (MyGet)
+
+on:
+  push:
+    branches:
+      - develop
+     
+env:
+  CONFIGURATION: Release
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
On `ubuntu` it's not able to build .net framework target, please check.

Reply:
As I understand correctly, from .NET5 it should work (I've tested in on my prive project). dotnet/sdk#4009